src/Reset.ts

```diff
@@ -299,7 +299,7 @@ const resetAddHistoryEntry = (input: resetNames, from = 'unknown') => {
         seconds: player.ascensionCounter,
         date: Date.now(),
         c10Completions: player.challengecompletions[10],
-        usedCorruptions: player.usedCorruptions.slice(0), // shallow copy,
+        usedCorruptions: player.corruptions.used, // shallow copy,
```

Comment:
```suggestion
        usedCorruptions: player.corruptions.used,
```
the comment no longer applies